src/natcap/invest/recreation/recmodel_client.py

```diff
@@ -1646,21 +1611,16 @@ def error_handler(err_level, err_no, err_msg):
         else:
             vector = gdal.OpenEx(path, gdal.OF_VECTOR)
             if vector is None:
-                raise ValueError(f"{path} did not load")
+                return f"{path} did not load"
             layer = vector.GetLayer()
             ref = osr.SpatialReference(layer.GetSpatialRef().ExportToWkt())
             layer = None
             vector = None
         if not base_ref.IsSame(ref):
-            LOGGER.warning(
-                f"{path} might have a different projection than the base AOI\n"
-                f"base:{base_ref.ExportToPrettyWkt()}\n"
-                f"current:{ref.ExportToPrettyWkt()}")
             invalid_projections = True
     if invalid_projections:
-        raise ValueError(
-            "One or more of the projections in the table did not match the "
-            "projection of the base vector")
+        return ("One or more of the projections in the table did not match "
+                "the projection of the base vector")
```

Comment:
Could we include `base_vector_path` in the message instead of saying "base vector"? I'm worried it's not obvious that "base vector" is the AOI vector. And maybe the same for using `table_path` instead of "table"?